docs/release/trg-0/trg-1-05.md

```diff
@@ -0,0 +1,35 @@
+---
+title: TRG 1.05 - Documentation Architecture
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft - MISSING EXTERNAL INFORMATION  | 10-Jan-2024  | Transfered from Release Acceptance Criteria into TRG|
+
+## Why
+
+It makes it much easier for others to use the component.
+>Who is the audience/adressant of the document?
+>```docs``` Folder
+
+## Description
+
+Arc42 documentation completed for the relevant product version.
+In Markdown format. Link to document available. Must point to the leading repository within Tractus-X.
+
+De-central:
+
+- MD files in Tractus-X repo.
+
+> specify Location of MD File
+
+## Best Practice
```

Comment:
Best practices should provide guidance on how to best fulfill/comply with the quality aspects of the TRG and not how to best engage in a review process.

Examples of potential best-practices:

- Provide each chapter of the Arc42 documentation as single `.md` file named after the chapter
- Favor diagrams as mermaid block in `.md` files directly over image files, where possible

Reply:
lines 32 & 33:
Provide each chapter of the Arc42 documentation as single .md file named after the chapter
Favor diagrams as mermaid block in .md files directly over image files, where possible
Have a look to peer repositories within Tractus-X for inspiration.